dataScrape/src/pokemonTableUpdate.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
```

Comment:
Is this being namespaced? I'm fairly certain everything else in our project will be namespaced so this should be as well.

Reply:
This part of the application is standalone, o does not need to be namespaced.